### DISCUSSION TITLE
when is the next release?

### DISCUSSION THREAD

**Discussion by tooptoop4** [tchiotludo/akhq#1087] [General]

**Comment by tchiotludo** [comment]
Why asking ? 
There is something that you wait on current dev branch ?

**Reply by r2ewj** [reply]
I would love to see the sort order preservation. :)

**Comment by tchiotludo** [comment]
It's now :smiley: 
https://github.com/tchiotludo/akhq/releases/tag/0.21.0
Hope you will enjoy that one :smile:
